src/DocumentPicker.Plugin/Shared/DocumentConverterService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using PdfToSvg;
+using Xamarin.Essentials;
+
+namespace DocumentPicker.Plugin.Shared
+{
+    public class DocumentConverterService : IDocumentConverterService
+    {
+        public string ConvertPdfToSvgString(DocumentResult documentResult)
+        {
+            using (var doc =  PdfDocument.Open(documentResult.FullPath))
+            {
+                return doc.Pages[0].ToSvgString();
+            }
+        }
+
+        public void ConvertPdfToSvg(DocumentResult documentResult, string outputSource)
+        {
+            using (var doc = PdfDocument.Open(documentResult.FullPath))
+            {
+                doc.Pages.First().SaveAsSvg(outputSource + "/" + documentResult.FileName.Split('.')[0] + ".svg");
+            }
+        }
+
+        public async Task<DocumentResult> PickDocumentAsync()
+        {
+            try
+            {
+                var customFileType =
+                    new FilePickerFileType(new Dictionary<DevicePlatform, IEnumerable<string>>
+                    { { DevicePlatform.iOS, new[] { "com.adobe.pdf" } }, // or general UTType values
+                        { DevicePlatform.Android, new[] { "application/pdf"} },
+                        { DevicePlatform.UWP, new[] { ".pdf"} }
+                    });
+
+                var options = new PickOptions()
+                {
+                    PickerTitle = "Please select a pdf file",
```

Comment:
please ensure the picker-title can be set from outside (as we need to translate to de, fr, etc.!)